server/src/main/java/io/spine/server/entity/FieldMasks.java

```diff
@@ -64,13 +64,13 @@ private FieldMasks() {
      * Applies the {@code FieldMask} to the given {@link Message}
      * if the {@code mask} parameter is valid.
      *
-     * <p>In case the {@code FieldMask} instance contains invalid field declarations,
+     * <p>If the {@code FieldMask} instance contains invalid field declarations,
      * they are ignored and do not affect the execution result.
      *
      * @param mask    the {@code FieldMask} to apply.
      * @param message the {@link Message} to apply given mask to.
-     * @return the message of the same type as the given one with only selected fields if
-     * the {@code mask} is valid, original message otherwise.
+     * @return the message of the same type as the given one with only selected fields. Otherwise if
```

Comment:
If think new text alters the original meaning. Also, the previous style follows the way it's done in standard [JDK Javadocs](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html).